app/src/ereferrals/java/org/eyeseetea/malariacare/data/database/datasources/ProgramLocalDataSource.java

```diff
@@ -24,5 +27,21 @@ public Program getUserProgram() {
         return program;
     }
 
+    @Override
+    public boolean checkLastDownloadedProgramMedia(String program) {
```

Comment:
I think that program associate to media should be program associated to the user then these methods are not necessary and remove media files is not a ProgramDataSource responsibility